EasyReflectometry/special/calculations.py

```diff
@@ -40,7 +40,7 @@ def neutron_scattering_length(formula: str) -> complex:
 def molecular_weight(formula: str) -> float:
     """
     Determine the molecular weight for a chemical formula.
-    
+
```

Comment:
Any reason to not use `periodictable` Formula object, with built-in parsers and molecular weight method?